packages/flutter/lib/src/material/tabs.dart

```diff
@@ -821,8 +822,16 @@ class TabBar extends StatefulWidget implements PreferredSizeWidget {
 
   /// The thickness of the line that appears below the selected tab.
   ///
-  /// The value of this parameter must be greater than zero and its default
-  /// value is 2.0.
+  /// The value of this parameter must be greater than zero.
+  ///
+  /// If [ThemeData.useMaterial3] is true and [TabBar] is a primary type,
```

Comment:
Can this section mention what is a primary TabBar and what is secondary (the secondary constructor)? If folks don't know, they may not be able to understand what this means.

Reply:
Great idea